database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2022.09-dev (Giant Rhubarb)
--- DB_UPDATE_VERSION 1469
+-- DB_UPDATE_VERSION 1470
```

Comment:
Since I just merged your previous PR, you need to increment these again.

Reply:
Do you want me to rebase this branch to `develop`?